evolver/settings.py

```diff
@@ -23,7 +23,7 @@ class Settings(BaseSettings):
 
 class AppSettings(BaseSettings):
     CONFIG_FILE: Path = Path("evolver.yml")  # in current directory
-    LOAD_FROM_CONFIG_ON_STARTUP: bool = True
+    LOAD_FROM_CONFIG_ON_STARTUP: bool = False
```

Comment:
I think here `False` is probably not a good idea, since this will need to be flipped to utilize any config that does get written (and typically that is desired). The more immediate fix would be to either instruct the user to copy a stub-out, or instruct them to start the first time as something like

```
EVOLVER LOAD_FROM_CONFIG_ON_STARTUP=false python -m evolver.app.main
```

then next time without. There is a deeper fix to edit the behavior altogether, but scope here could probably be just the startup docs as-is

Reply:
I second this, setting this to ``False`` for this reason doesn't seem correct as they'd only want to turn it back on after bootstrapping. I think this should be ``True`` by default.

Reply:
With this change, this pr is also in conflict/dupe of #141